### PR TITLE
fix(core): add key control to scrollbar

### PR DESCRIPTION
**Pull request by shrvr**
## Related Issue(s)

part of #8557 

## Description

Solves [this](https://github.com/SAP/fundamental-ngx/issues/8557#issuecomment-1215412466): Added arrow key control to scrollbar.

### CODE REVIEW

**Review by mikerodonnell89 — COMMENTED** [review]
Works well but I wonder if this is the focus outline we want when tabbing to the container? Doesn't happen when clicking in. Not seeing anything in the designs @InnaAtanasova any thoughts
Screen Shot 2022-10-04 at 9 56 20 AM

**Comment by shrvr** [comment]
> Works well but I wonder if this is the focus outline we want when tabbing to the container? Doesn't happen when clicking in. Not seeing anything in the designs @InnaAtanasova any thoughts Screen Shot 2022-10-04 at 9 56 20 AM

I think that's  the default focus `outline` by browser. We can set it to `none` for Scrollbar component in fd-styles. Let me know your thoughts. @mikerodonnell89  @InnaAtanasova

File: libs/core/src/lib/scrollbar/scrollbar.component.ts [hunk 1]
```diff
@@ -54,6 +54,9 @@ export class ScrollbarComponent {
         return this._alwaysVisible;
     }
 
+    @HostBinding('attr.tabindex')
+    tabindex = 0;
```
Comment (platon-rov) [inline libs/core/src/lib/scrollbar/scrollbar.component.ts:58]: typedocs or `@hidden` and `_` prefix.

**Review by platon-rov — APPROVED** [review]
place make scrollbar area focus outline is accordingly to the specs (apply focus mixin to the comp in styles).

**Pull request merged**